app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+  has_many :likes
```

Comment:
Please add foreign keys to each, check out [active record associations](https://guides.rubyonrails.org/association_basics.html).